### ISSUE TITLE
LINK please

### ISSUE THREAD

**Issue by akocart07** [nicehash/NiceHashMiner-Archived#1131]
Is there list of GPU can we used in mining.... specifically here in NICEHASH.com?????? so we will know what kind of GPU that are not allowed . thanks

**Comment by Arcandel** [comment]
https://www.nicehash.com/?p=calc  you should be safe with those on the list

**Comment by OneOfMany07** [comment]
Yeah, guessing that list would be VERY long.  I'd start with CUDA compatible cards if NVidia oriented.  No idea about AMD valid cards to use.

https://developer.nvidia.com/cuda-gpus
